internal/git/git.go

```diff
@@ -183,13 +184,12 @@ func (s *Service) LocateRelease(ctx context.Context, r *git.Repository, artifact
 }
 
 func locateReleaseCondition(artifactID string) conditionFunc {
-	r := regexp.MustCompile(fmt.Sprintf(`(?i)release %s($|\r\n|\r|\n|\sby\s\S+)`, regexp.QuoteMeta(artifactID)))
-	return func(commitMsg string) bool {
-		if artifactID == "" {
-			return false
-		}
-		return r.MatchString(commitMsg)
+	if artifactID == "" {
+		return falseConditionFunc
 	}
+	return commitinfo.LocateRelease(func(c commitinfo.CommitInfo) bool {
+		return strings.ToLower(c.ArtifactID) == strings.ToLower(artifactID)
```

Comment:
This condition can be simplified (and optimized, if thats your thing) with [`strings.EqualFold`](https://golang.org/pkg/strings/#EqualFold). :) 
```suggestion
		return strings.EqualFold(c.ArtifactID, artifactID)
```

Reply:
Didn't know that, and haven't heard the term "EqualFold" before, but sounds delightful 💪